libkbfs/kbfs_ops_test.go

```diff
@@ -4479,7 +4479,8 @@ func TestKBFSOpsBackgroundFlush(t *testing.T) {
 
 	// start the background flusher
 	ops := getOps(config, rootNode.GetFolderBranch().Tlf)
-	go ops.backgroundFlusher(1 * time.Millisecond)
+	config.SetBGFlushPeriod(1 * time.Millisecond)
```

Comment:
I agree -- this test was written before I knew about clockwork, and I didn't have an easy way to use our fake clock with the `time` package.  But since this is an unrelated pre-existing condition, I'm going to leave that change out of this PR.